src/hooks/useSwapOptions.ts

```diff
@@ -65,11 +63,12 @@ function useSwapOptions(
               debouncedTo,
               debouncedIsBuyOrder ? undefined : parseUnits(debouncedValue, debouncedFrom.decimals),
               debouncedIsBuyOrder ? parseUnits(debouncedValue, debouncedTo.decimals) : undefined,
-              debouncedSorting,
+              'most-profit',
```

Comment:
we should probably drop this into a constant in `src/constants/aggregator.ts` file

Reply:
Sounds good!

Reply:
Fixed